paddle/math/CpuSparseMatrix.h

```diff
@@ -43,6 +43,11 @@ class CpuSparseMatrix : public Matrix {
 
   MatrixPtr getTranspose();
 
+  MatrixPtr getInverse() { LOG(FATAL) << "Not implemented"; return NULL; }
```

Comment:
It's defined as a pure virtual function in its base class. It I do not define it here, I cannot allocate an object of abstract type.